src/test/emails.spec.ts

```diff
@@ -203,5 +206,12 @@ describe('emails', () => {
       const email = await renderApplicationPausedEmail(app, uiLinksStub, configStub);
       console.log(email.emailMjml);
     });
+
+    it('should render an attestation received email', async () => {
```

Comment:
should this have some kind of assertion? If a function runs even "incorrectly" I think that's a passing test without any assertions

Reply:
these tests are mostly just to generate test email mjml to make sure it looks right rather than pass/fail